app/containers/ConfirmBattleContainer.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react'
+import ConfirmBattle from '../components/ConfirmBattle'
+import { getPlayersInfo } from '../utils/githubHelpers'
+
+class ConfirmBattleContainer extends Component {
+  constructor (props, context) {
+    super (props, context)
```

Comment:
the space here is inconsistent. you normally have a space after a function definition, but not a call.